src/main/java/com/cs544/project/domain/CourseRegistration.java

```diff
@@ -17,4 +17,8 @@ public class CourseRegistration {
     @ManyToOne
     @JoinColumn(name = "CourseOfferingId")
     private CourseOffering courseOffering;
+
+    @OrderColumn(name="Sequence")
+    @Column(name="Sequence")
+    int sequence;
```

Comment:
this shouldn't be there since we won't be using sequence.
normally it should be @OrderColumn(name="sequence")